src/main/java/entities/default_game/Player.java

```diff
@@ -116,4 +137,43 @@ public void setStageClear(boolean b) {
     public boolean getStageClear(){
         return stageClear;
     }
+
+    /** Set the player's direction */
+    public void setDirection(String direction){
+        this.direction = direction;
+    }
+
+    /** Get the player's direction */
+    public String getDirection(){
+        return direction;
+    }
+    /** Draw the player's sprite according to its current direction */
+    public void draw(IDrawOutputBoundary d) {
+        Graphics2D g2 = d.graphics();
+        int tileSize = d.getTileSize();
+        int xPixels = getX() * tileSize;
+        int yPixels = getY() * tileSize;
+        BufferedImage image = null;
+        switch (getDirection()) {
+            case "up" -> {
```

Comment:
Looks good! You might want to change all the -> here since it's causing the autograding to fail